### PR TITLE
ci: add scripts for storybook deployment

### PR DESCRIPTION
**Pull request by sutarmin**
This PR adds an ability to manually deploy storybook to gh-pages: https://devexperts.github.io/dx-platform .

### CODE REVIEW

File: packages/react-kit/package.json [hunk 1]
```diff
@@ -10,7 +10,10 @@
     "clean": "dx-tools clean dist",
     "prepare": "yarn clean && yarn build",
     "start": "dx-tools storybook",
-    "watch": "dx-tools build-lib src dist -w"
+    "watch": "dx-tools build-lib src dist -w",
+    "storybook:build": "build-storybook -o .out -c ../../node_modules/@devexperts/tools/dist/config/storybook",
+    "storybook:deploy": "storybook-to-ghpages --ci --existing-output-dir=.out --host-token-env-variable=GITHUB_TOKEN",
+    "storybook:publish": "yarn storybook:build && yarn storybook:deploy"
```
Comment (raveclassic) [inline packages/react-kit/package.json]: Let's integrate into travis deployment

Comment (raveclassic) [reply packages/react-kit/package.json]: On master branch

**Comment by sutarmin** [comment]
Added conditional build step for Travis. I'm not sure if this will work though, only merging will show. 

The coding part is ready for review but we need to issue github token with safe enough permissions and put it into travis environment before merging the PR.

File: packages/react-kit/package.json (hunk 1 above)
Comment (sutarmin) [reply packages/react-kit/package.json]: Done

**Comment by sutarmin** [comment]
I've added token to Travis env, now this PR is ready to be merged. I hope it will start up on the first try

**Pull request merged**